packages/components/src/DateRangePicker/_docs/DateRangePicker.stories.tsx

```diff
@@ -58,3 +58,99 @@ export const Playground: Story = {
     },
   },
 }
+
+const selectedDateRange = {
+  from: new Date(2022, 2, 6),
+  to: new Date(2022, 2, 15),
+}
+
+export const AboveIfAvailable: Story = {
```

Comment:
Same kind of stories I added for the DatePicker - this should cover us for testing the open state against the axe tool. These are also included in the MDX as a reference so I haven't bumped them to a spec.stories file.